argnorm/drug_categorization.py

```diff
@@ -15,45 +15,34 @@ def get_immediate_drug_classes(aro_num: str) -> list[list]:
         aro_num (str): ARO number. Needs to be in the form 'ARO:number'.
 
     Returns:
-        drug_classes_list (list[list]): 
+        drug_classes_list (list[tuple]): 
             A two-dimensional list where each inner list represents a drug class.
             Each inner list contains the ARO number and name of the drug class in that order. [ARO:number, name].
     '''
 
-    GENE = ARO[aro_num]
-    confers_resistance_to_drug_class = False
-    confers_resistance_to_antibiotic = False
+    gene = ARO[aro_num]
 
-    if confers_resistance_to_drug_class == False or confers_resistance_to_antibiotic == False:
-        for i in GENE.relationships:
-            if i.name == 'confers_resistance_to_drug_class' and confers_resistance_to_drug_class == False:
-                confers_resistance_to_drug_class = True
-
-            if i.name == 'confers_resistance_to_antibiotic' and confers_resistance_to_antibiotic == False:
-               confers_resistance_to_antibiotic = True
+    confers_resistance_to_drug_class = any(r.name == 'confers_resistance_to_drug_class' for r in gene.relationships)
+    confers_resistance_to_antibiotic = any(r.name == 'confers_resistance_to_antibiotic' for r in gene.relationships)
 
     drug_classes = []
 
     if confers_resistance_to_drug_class:
-        for drug_class in GENE.relationships[ARO.get_relationship('confers_resistance_to_drug_class')]:
-            drug_classes.append(drug_class)
+        for drug_class in gene.relationships[ARO.get_relationship('confers_resistance_to_drug_class')]:
+            drug_classes.append([drug_class.id, drug_class.name])

```

Comment:
This is not a `tuple`

Reply:
Yes, I missed it in the commit. I've added the tuples in this file with the changes to the smoke tests (i.e using the zip function) now.